src/components/Homepage/HomepageHowItWorks.vue

```diff
@@ -86,10 +97,9 @@ export default {
 @import 'settings';
 
 .how-it-works {
-	padding: 2rem 0;
-
 	&__list {
 		list-style: none;
+		justify-content: space-around;
```

Comment:
This ensures that if we have fewer than 4 list items they distribute nicely when not stacked.